scripts/securedrop-admin.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+"""
+Admin wrapper script for applying salt states for staging and prod scenarios. The rpm
+packages only puts the files in place `/srv/salt` but does not apply the state, nor
+does it handle the config.
+"""
+import sys
+import argparse
+import subprocess
+import os
+from validate_config import SDWConfigValidator, ValidationError
+
+SCRIPTS_PATH = "/usr/share/securedrop-workstation-dom0-config/"
+SALT_PATH = "/srv/salt/sd/"
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--apply",
+        default=False,
+        required=False,
+        action="store_true",
+        help="Apply workstation configuration with Salt",
+    )
+    parser.add_argument(
+        "--validate",
+        default=False,
+        required=False,
+        action="store_true",
+        help="Validate the configuration",
+    )
+    args = parser.parse_args()
+
+    return args
+
+
+def copy_config():
+    """
+    Copies config.json and sd-journalist.sec to /srv/salt/sd
+    """
+    try:
+        subprocess.check_call(
+            ["sudo", "cp", os.path.join(SCRIPTS_PATH, "config.json"), SALT_PATH]
+        )
+        subprocess.check_call(
+            ["sudo", "cp", os.path.join(SCRIPTS_PATH, "sd-journalist.sec"), SALT_PATH]
+        )
+    except subprocess.CalledProcessError as e:
+        print(str(e))
+        sys.exit(1)
```

Comment:
Why catch, then sys.exit? If we simply allow the exception to be raised, we'll still have the script halt execution, as well as exit non-zero. Does the syntax here prevent preventing the traceback? (Even if it does, not sure it's worth the extra lines.)